src/AutoMapper/TypeMap.cs

```diff
@@ -105,7 +105,7 @@ public int MaxDepth
 
         public PropertyMap[] GetPropertyMaps()
         {
-            return _sealed ? _orderedPropertyMaps : _propertyMaps.Concat(_inheritedMaps).ToArray();
+            return _orderedPropertyMaps ?? _propertyMaps.Concat(_inheritedMaps).ToArray();

```

Comment:
I'm not sure this is the best place, there's a ctor in PropertyMap that sets all the inherited properties (if they're not already set).